etc/korrel8r/rules/netflow.yaml

```diff
@@ -0,0 +1,10 @@
+rules:
+  - name: NetflowToPod
+    start:
+      domain: netflow
+    goal:
+      domain: k8s
+      classes: [Pod]
```

Comment:
Just guessing, but as the query may have Pod, Service or Node via `SrcK8S_Type`, would that work to have classes here as `[Pod, Service, Node]` ?

Reply:
Yep, this rule can have all the resources that can appear as SrcK8S_Type as goals. Can you provide a complete list?

Reply:
This is the complete list for SrcK8S_Type (Pod, Service, Node), however more classes can be managed with SrcK8S_OwnerType (Deployment, DaemonSet, etc.) ; in netobserv this is an open list, since it is populated under the hood by the "OwnerReference" of pods / replicasets, which can be anything. For example, with KubeVirt we can see Virtual Machines as SrcK8S_OwnerType. I guess we can stick with the most common ones